packages/connections/src/components/connecting/connecting-animation.spec.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+import ConnectingAnimation from './connecting-animation';
+
+describe('ConnectingAnimation Component', function () {
+  describe('when rendered', function () {
+    let unmountComponent;
+    let cancelAnimationFrameSpy;
+    let requestAnimationFrameSpy;
+    let timeoutCode;
+
+    beforeEach(function () {
+      if (!window.requestAnimationFrame) {
+        window.requestAnimationFrame = (requestedPaintFunc) => {
+          timeoutCode = setTimeout(requestedPaintFunc, 5);
+          return 3;
+        };
+        window.cancelAnimationFrame = (codeToCancel) => {
+          expect(codeToCancel).to.equal(3);
+          if (timeoutCode) {
+            clearTimeout(timeoutCode);
+          }
+          timeoutCode = null;
+        };
+      }
+
+      requestAnimationFrameSpy = sinon.spy(window.requestAnimationFrame);
+      cancelAnimationFrameSpy = sinon.spy(window.cancelAnimationFrame);
+
+      sinon.replace(window, 'requestAnimationFrame', requestAnimationFrameSpy);
+      sinon.replace(window, 'cancelAnimationFrame', cancelAnimationFrameSpy);
+
+      const { unmount } = render(<ConnectingAnimation />);
+      unmountComponent = unmount;
+    });
+    afterEach(function () {
+      unmountComponent = null;
+      sinon.restore();
+    });
+
+    it('calls to request an animation frame', function () {
+      expect(requestAnimationFrameSpy.called).to.equal(true);
+    });
+
+    it('does not call to cancelAnimationFrame', function () {
+      expect(cancelAnimationFrameSpy.called).to.equal(false);
+    });
```

Comment:
Yeah we probably don't need to test it, I think I mostly did it since I haven't used hooks too much and it made me feel a bit more re-assured. If it's alright I'll leave it in just to protect against unintended change of behavior.

Reply:
Yeah, makes sense to have a confidence boost like that once in a while 😄 Also not that big of a deal to keep it around at all